library/Faktory/Worker.hs

```diff
@@ -152,7 +150,7 @@ runWorker
   -> IO ()
 runWorker settings workerSettings handler = do
   worker <- startWorker settings workerSettings handler
-  waitUntilDone worker
+  void $ waitUntilDone worker
```

Comment:
Why not return the exception here?  It seems like something the caller might want to know about.

Reply:
`runWorker` is what the original library uses - I wanted to keep the type signatures the same. We could probably drop the function since we dont use it, but it'd require updating some more code